rest-client/pom.xml

```diff
@@ -154,6 +154,30 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
This will create two more jars, so the artifacts generated are
- rest-client-0.12.27.pom
- rest-client-0.12.27.jar
- rest-client-0.12.27-javadoc.jar
- rest-client-0.12.27-sources.jar

It's a convention maven follows. I've tested this with `mvn install` and will check travis to see if deploy works (don't see why it wouldn't)